traffic_control/utils/send_traffic_sign_reals.py

```diff
@@ -0,0 +1,77 @@
+import argparse
+import os
+
+import requests
+from osgeo import ogr
+from requests.auth import HTTPBasicAuth
+
+SOURCE_SRID = 3879
+SOURCE_NAME = "BLOM streetview2019"
+OWNER = "Helsingin kaupunki"
+
+parser = argparse.ArgumentParser(
+    description="Send traffic sign real data to the platform from SHP-file. "
+    "Note that SHP-metadata and related files need to also exist in the same folder."
+)
+parser.add_argument(
+    "--username", required=True, type=str, help="Admin-username to make the import"
+)
+parser.add_argument(
+    "--password", required=True, type=str, help="Admin-password to make the import"
+)
+parser.add_argument(
+    "--url",
+    required=True,
+    type=str,
+    default="http://localhost:8000/fi/api/traffic-sign-reals/",
+    help="API-endpoint url where to post the data",
+)
+parser.add_argument(
+    "--filename",
+    required=True,
+    type=str,
+    help="Path to the traffic sign reals shp file",
+)
+
+args = parser.parse_args()
+
+filename = args.filename
+
+if not os.path.exists(filename):
+    raise Exception("File {0} does not exist".format(filename))
+
+file = ogr.Open(filename)
+layer = file.GetLayer(0)
+print("SHP-file has {0} features.".format(layer.GetFeatureCount()))
+auth = HTTPBasicAuth(args.username, args.password)
+counter = 0
+layer.ResetReading()
+for feature in layer:
```

Comment:
It's a good practice to add an empty line before a bigger block like this `for`.

Reply:
Will do